### DISCUSSION TITLE
Slate AX at 94% RAM usage -- is that a problem?

### DISCUSSION THREAD

**Discussion by MikeHa** [SmoothWAN/SmoothWAN#146] [General & Help]
Total RAM available is 400MB I believe.  We have had moments of slow downs and I don't know if it's related.  Would an RP4 be a better option?

**Comment by TalalMash** [comment]
This is the third report so far, definitely looks like an issue, can you go to Services -> Terminal and paste `htop -s PERCENT_MEM` then post a screenshot to show if it's process related?

**Comment by TalalMash** [comment]
Flint/Slate AX patch for testing, in terminal, type:
Slate:
 ```
wget https://raw.githubusercontent.com/SmoothWAN/SmoothWAN/main/devconfigs/slateax/files/etc/rc.local -O /etc/rc.local
```
Flint:
 ```
wget https://raw.githubusercontent.com/SmoothWAN/SmoothWAN/main/devconfigs/flint/files/etc/rc.local -O /etc/rc.local
```
